src/main/java/frc/robot/RobotContainer.java

```diff
@@ -179,16 +180,28 @@ private void configureButtonBindings() {
 	 * @return the command to run in autonomous
 	 */
 	public Command getAutonomousCommand() {
-		// Two ball autonomous routine.
-		return new SequentialCommandGroup(
-				new ArmCommand(m_shooterSubsystem, ShooterConstants.kLowerArmAngle),
+		String path = SmartDashboard.getString("Autonomous Path", "BlueMid");
+		SequentialCommandGroup twoBallAuton = new SequentialCommandGroup(
 				new ParallelCommandGroup(
-						new PathWeaverCommand(m_swerveDriveSubsystem,
-								SmartDashboard.getString("Autonomous Path", "BlueHangar") + "TwoBall1", true),
-						new IntakeCommand(m_shooterSubsystem)),
-				new ArmCommand(m_shooterSubsystem, ShooterConstants.kUpperArmAngle),
-				new PathWeaverCommand(m_swerveDriveSubsystem,
-						SmartDashboard.getString("Autonomous Path", "BlueHangar") + "TwoBall2", false),
+						new PathWeaverCommand(m_swerveDriveSubsystem, path + "TwoBall1", true),
+						new SequentialCommandGroup(
+								new ArmCommand(m_shooterSubsystem, ShooterConstants.kLowerArmAngle),
+								new IntakeCommand(m_shooterSubsystem, 1))),
+				new ParallelCommandGroup(
+						new ArmCommand(m_shooterSubsystem, ShooterConstants.kUpperArmAngle),
+						new PathWeaverCommand(m_swerveDriveSubsystem, path + "TwoBall2", false)),
+				new ShootCommand(m_shooterSubsystem));
+		SequentialCommandGroup fourBallAuton = new SequentialCommandGroup(
+				twoBallAuton,
+				new ParallelCommandGroup(
+						new PathWeaverCommand(m_swerveDriveSubsystem, path + "FourBall3", false),
+						new SequentialCommandGroup(
+								new ArmCommand(m_shooterSubsystem, ShooterConstants.kLowerArmAngle),
+								new IntakeCommand(m_shooterSubsystem, 1))),
+				new ParallelCommandGroup(
+						new PathWeaverCommand(m_swerveDriveSubsystem, path + "FourBall4", false),
+						new ArmCommand(m_shooterSubsystem, ShooterConstants.kUpperArmAngle)),
 				new ShootCommand(m_shooterSubsystem));
+		return fourBallAuton;
```

Comment:
we actually want to use parallel deadline group instead of parallel command group so that we can move on to the next path without waiting for the intake to finish running. This saves a little bit of time. If your worried that the intake isn't being run long enough to properly index the balls you could add another intake command that runs during the next path